packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -108,14 +135,6 @@ export const menuItemStyles = (context, definition) =>
         justify-content: center;
     }
     
-    ::slotted(svg) {
-        ${
-            /* Glyph size and margin-left is temporary - 
-            replace when adaptive typography is figured out */ ""
-        } width: 16px;
-        height: 16px;
-    }
```

Comment:
I know where the removal of this came from, but I don't think we should make that change *just* here. If we're planning on making updates to accommodate both 20x20 and 16x16 SVG's we should do that in a separate branch as an entire feature. Otherwise, we are piecemeal removing things without fully validating the work. Can we add this back in and open an issue to track the update for 20x20 icons?